src/plugins/intel_cpu/src/cpu_memory.cpp

```diff
@@ -35,17 +35,24 @@ namespace {
 
     void transferData(const IMemory& src, const IMemory& dst, bool ftz) {
         node::Reorder::reorderData(src, dst);
+        try {
+            auto src_data_type = DnnlExtensionUtils::IEPrecisionToDataType(src.getDesc().getPrecision());
```

Comment:
`IMemory` should have `ov::element::Type` inside and we should avoid legacy precisions and other structures.

Reply:
sure, will update per Maksim suggestion.